### ISSUE TITLE
Input of color_temp?

### ISSUE THREAD

**Issue by station7** [mabunixda/node-red-contrib-alexa-home#121]
I love your Alexa implementation for node-red - many thanks for your efforts! I just have a very quick question:

I have not seen a description of the payload for the input of color_temp values ("_At the moment you can input as payload objects:
brightness color on/off_").  Is the input of ct-values planned or maybe already implemented and I did not recognize it?

**Comment by station7** [comment]
And another short question: Regardless of what type of light I choose (dimmable, color temperature or extended) the Alexa-App for iOS allows me to change the color temperature or set a color for each light - even for dimmable lights only. Is this behaviour a limit of Amazon/the Alexa App?

**Comment by mabunixda** [comment]
Yeah the color setting seems to be buggy #56 :( Fixxes/Help is welcome
